src/Exception/FilesystemException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace App\Exception;
+
+class FilesystemException extends \Exception
```

Comment:
```suggestion
class FilesystemException extends AppException
```

(after rebase with `develop`).